bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/creators/ValueCreator.java

```diff
@@ -768,6 +771,9 @@ public static BMap<BString, Object> createRecordValue(RecordType recordType) {
     /**
      * Create a runtime record value with given initial values and given record type.
      *
+     * <p>To create a record value using a record type defined in Ballerina,
+     * do not use this method and use another createRecordValue method.
+     *
```

Comment:
```suggestion
     * <p>This method creates a record value using a record type defined in Ballerina. 
     * Please use the other @link{ #createRecordValue()} methods instead of this.
     *
```

Reply:
This method is used to create a BMap from given java instance of record type which can only be used inside java native code.  If the recordType defined in the ballerina code and the created map value needs to be used in the ballerina, Please use createRecordValue apis with packageId, recordTypeName to create a Map.